datasets/us/ss_most_wanted/us_ss_most_wanted.yml

```diff
@@ -0,0 +1,15 @@
+title: "United States Secret Service Most Wanted Fugitives"
```

Comment:
let's start adding data assertions to all new crawlers - it makes reviews easier. I also add them when I make significant changes to an existing crawler almost like test-driven development https://zavod.opensanctions.org/metadata/#data-assertions

Also add https://zavod.opensanctions.org/metadata/#data-coverage and `load_db_uri`

So I think what this tells me is we need to add `load_db_uri`, `coverage` and `assertions` to https://zavod.opensanctions.org/best_practices/merge_checklist/ ?

Reply:
I think it would be good to add to the checklist, I found it very helpful. For instance, I only noticed it wasn't emitting the name because I read through the checklist.

Reply:
@jbothma I tried the assertion with wrong values but did not get any warnings for some reason.
Can you please see if I am doing something wrongly?

Thank you

Reply:
oh good point - the warnings should occur when you run `zavod validate --clear`
I'll update the documentation

Reply:
Got it... Thanks